src/strategies/aks/index.ts

```diff
@@ -0,0 +1,104 @@
+import fetch from 'cross-fetch';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'akshay';
```

Comment:
Hi Chaitanya,

Thanks for the quick response. 

Actually Cake strategy is by Pancakeswap which a completely independent organisation and has a good amount of user traffic. If we make changes in that strategy (like adding extra params), it might break the things for them.

That's why I created an independent strategy which is a replica for cake strategy.

Reply:
I've committed the changes suggested by you.